src/shared/pipes/modifyLabel.ts

```diff
@@ -0,0 +1,25 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'modifyLabel',
+})
+export class ModifyLabel implements PipeTransform {
+
+  /**
+   * Pipe to convert competency label dynamically
+   * @param value
+   * @param isBikeCategory
+   * @param fault
+   */
+  transform(value: string, isBikeCategory: boolean, fault: string): string {
+    // For CatA tests change move off to move away
+    if (isBikeCategory && (fault === 'moveOffSafety' ||
```

Comment:
could maybe simplify this a little, e.g. 
 if (isBikeCategory && fault.toUpperCase().replace(' ','').includes('MOVEOFF'))

Reply:
dont think i can do it like that....that wouldnt cover all 4 'OR's. on the office page it passes in the string (evaluated in the ts where i cant apply a pipe).   moveOffSafety, moveOffControl would be fine, but 'Move off - Safety' and 'Move off - Control' wouldnt.

Reply:
changed to fault.toUpperCase().replace(/\s/g, '').includes('MOVEOFF')).  works